lib/awesome_spawn/null_logger.rb

```diff
@@ -0,0 +1,11 @@
+unless defined?(NullLogger)
+  require 'logger'
+
+  class NullLogger < Logger
```

Comment:
This should be in the AwesomeSpawn namespace.  Then you don't need the unless defined? check.